### PR TITLE
update links

### PR DESCRIPTION
**Pull request by Arhell**
resolve https://github.com/kubernetes-sigs/reference-docs/issues/289

### CODE REVIEW

File: genref/output/md/kubelet-config.v1beta1.md [hunk 1]
```diff
@@ -897,7 +897,7 @@ managers are running. Valid values include:</p>
    <p>systemReserved is a set of ResourceName=ResourceQuantity (e.g. cpu=200m,memory=150G)
 pairs that describe resources reserved for non-kubernetes components.
 Currently only cpu and memory are supported.
-See http://kubernetes.io/docs/user-guide/compute-resources for more detail.
+See https://kubernetes.io/docs/user-guide/compute-resources for more detail.
```
Comment (tengqm) [inline genref/output/md/kubelet-config.v1beta1.md:900]: This file is auto-generated, checked in for caching.
Please propose your fix to the upstream if needed.

Comment (Arhell) [reply genref/output/md/kubelet-config.v1beta1.md]: updated, where can I fix the links so that it pulls up in the generated files?

Comment (tengqm) [reply genref/output/md/kubelet-config.v1beta1.md]: https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kubelet/config/v1beta1/types.go#L599
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kubelet/config/v1beta1/types.go#L769
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kubelet/config/v1beta1/types.go#L785

**Pull request merged**